src/backends/torch/torchlib.cc

```diff
@@ -606,46 +606,48 @@ namespace dd
             = ad_mllib.has("rotate") && ad_mllib.get("rotate").get<bool>();
         this->_logger->info("rotate: {}", has_rotate);
         int crop_size = -1;
+        CropParams crop_params;
         if (ad_mllib.has("crop_size"))
           {
             crop_size = ad_mllib.get("crop_size").get<int>();
+            crop_params
+                = CropParams(crop_size, inputc.width(), inputc.height());
             this->_logger->info("crop_size : {}", crop_size);
           }
         float cutout = 0.0;
+        CutoutParams cutout_params;
         if (ad_mllib.has("cutout"))
           {
             cutout = ad_mllib.get("cutout").get<double>();
+            cutout_params
+                = CutoutParams(cutout, inputc.width(), inputc.height());
```

Comment:
```cpp
            cutout_params
                = CutoutParams(ad_mllib.get("cutout").get<double>(), inputc.width(), inputc.height());
            this->_logger->info("cutout: {}", cutout_params.cutout);
```
And remove `float cutout = 0.0`? Same  for `crop_size`.

Reply:
done